### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Richa-rani02**
Added Login/Home Screen (no functionality)
signup Modal (no functionality)
AvatarModal
Note:commented code is for further implementation purpose

### CODE REVIEW

File: src/components/SignupModal/SignupModal.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return(
+        <Modal isOpen={isOpen} onClose={onClose}>
+             <div className="px-0-25 flex-col flex-center">
+        <div className="logo">
+          <img src="../Assets/logosm.png"/>
+        </div>
+        <h2 className="login-page__login-title">Signup</h2>
+        {/* <div className="error-msg mt-1 px-0-75 py-0-5">
+          Error in Signup
+        </div> */}
```
Comment (nerdyspook) [inline src/components/SignupModal/SignupModal.jsx:13]: You can remove the comment from here if its not required in future.

File: src/pages/auth/Login.jsx [hunk 2]
```diff
@@ -1,7 +1,40 @@
...
+            <div className="mt-1 signup-link" onClick={() => setSignupActive(true)}>New user ? SignUp here</div>
+            <button className="auth-btn-group auth-btn py-0-75 mt-2">
+              Login
+            </button>
+            <div className="m-1 test-login">Load test credential</div>
+          </form>
+        </div>
+      </div>
+      {signupActive ? <AvatarModal isOpen={signupActive} onClose={handleSignupToogle} /> : null}
+      {/* {signupActive ? <SignupModal isOpen={signupActive} onClose={handleSignupToogle} /> : null} */}
```
Comment (nerdyspook) [inline src/pages/auth/Login.jsx:37]: You can remove the comment from here as well.

**Review by nerdyspook — APPROVED** [review]
Code looks well structured and clean.

**Review by Mohitkumar33 — APPROVED** [review]
clean code

File: src/components/SignupModal/SignupModal.jsx (hunk 1 above)
Comment (Richa-rani02) [reply src/components/SignupModal/SignupModal.jsx:13]: will remove it after further implementation

**Pull request merged**